StreamChatClient_v3/Controllers/ChannelController.swift

```diff
@@ -0,0 +1,27 @@
+//
+// ChannelController.swift
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+/// `ChannelController` allows observing and mutating the controlled channel.
+///
+///  ... you can do this and that
+///
+public class ChannelController {
```

Comment:
Why don't you
```swift
protocol Controller: class {
  let client: ChatClient
}
```
as you did with `Reference`s in AirChat?

Reply:
I didn't see a reason for that yet 🤷‍♂️